aspnetcore/blazor/get-started.md

```diff
@@ -139,30 +143,30 @@ Each time the **Click me** button is selected:
 
 The runtime compares the new content to the previous content and only applies the changed content to the Document Object Model (DOM).
 
-Add a component to another component using an HTML-like syntax. Component parameters are specified using attributes or child content. For example, a Counter component can be added to the app's homepage by adding a `<Counter />` element to the Index component.
+Add a component to another component using an HTML syntax. Component parameters are specified using attributes or child content. For example, a Counter component can be added to the app's homepage by adding a `<Counter />` element to the Index component.
 
 *Pages/Index.razor*:
 
 [!code-cshtml[](get-started/samples_snapshot/3.x/Index1.razor?highlight=7)]
 
-Run the app. The homepage has its own counter.
+Run the app. The homepage has its own counter provided by the Counter component.
 
 To add a parameter to the Counter component, update the component's `@functions` block:
 
-* Add a property for `IncrementAmount` decorated with the `[Parameter]` attribute.
+* Add a property for `IncrementAmount` with an `[Parameter]` attribute.
```

Comment:
a `[Parameter]` attribute?

Reply:
yep ....... would you believe that the "n" key on my MS ergo keyboard sticks and randomly inserts "n"'s into the topics????? ........ no .............. that's my story, and I'm sticking with it. 🙈